speeduino/utils.h

```diff
@@ -6,6 +6,7 @@ These are some utility functions and variables used through the main code
 
 #include <Arduino.h>
 
+#define CONCATS(s1, s2) (s1" " s2) //needed for some reason. not defined correctly because of utils.h file of speeduino (same name as one in arduino core)
```

Comment:
When using the CDC usb serial parts of the stm32 core some weird things happen with the defines. It has to do something with the util.h in the core defining __UTILS_H  and speeduino defines something like UTILS_H. This breaks compilation with usb CDC serial enabled. This line fixes it for now.